package/main.yaml

```diff
@@ -1,9 +1,10 @@
 applications:
   - appcat
   - crossplane
+  - appuio-reporting as appcat-reporting
 parameters:
   pkg.appcat:
-    componentVersion: v2.13.1
+    componentVersion: v2.14.0
```

Comment:
This is actually not necessary, as the Github Action will set the correct version automatically.

Reply:
Huh TIL, thanks.